client/signup/controller.js

```diff
@@ -112,6 +114,25 @@ export default {
 					const countryCode = geo.data.body.country_short;
 					if ( 'gutenberg' === abtest( 'newSiteGutenbergOnboarding', countryCode ) ) {
 						window.location.replace( window.location.origin + '/new' + window.location.search );
+					} else if (
```

Comment:
Did your flow need `removeWhiteBackground()` that it now skips due never entering `else` block?